include/deal.II/grid/grid_generator.h

```diff
@@ -2424,6 +2424,25 @@ namespace GridGenerator
                                     Triangulation<1, spacedim>       &out_tria);
 #endif
 
+
+  /**
+   * Perform an Alfeld split (also called barycentric refinement) of a simplex
+   * mesh.
+   *
+   * This function takes a simplex mesh (@p in_tria) and subdivides each
+   * triangle into three triangles with a single new vertex (the barycenter). In
+   * the process, the simplex mesh is flattened (no hierarchy is kept).
```

Comment:
Perhaps add a picture and mention the Scott-Vogelius connection?

Reply:
Separately, the documentation mentions triangles and simplices. Is it only in 2d, or also in 3d?